modules/carto/src/sources/index.ts

```diff
@@ -0,0 +1,32 @@
+export {SOURCE_DEFAULTS} from './common';
+export type {CartoTilejsonResult, GeojsonResult, JsonResult} from './common';
+
+export {CartoH3QuerySource} from './h3-query-source';
```

Comment:
@felixpalmer the functions in JS should be in camelCase instead of PascalCase, can you change them? 

You should review all of them.

Thanks

Reply:
OK, fair enough, I'll rename to `cartoXXXSource` etc. I didn't want to implement these as classes, as they are essentially just wrappers around `fetch()`

Reply:
Great, thx!